pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -760,7 +760,7 @@ func (m *migrationMonitor) processInflightMigration(dom cli.VirDomain, stats *li
 
 		progressDelay := now - m.lastProgressUpdate
```

Comment:
Didn't look to closely but isn't the change from 648 also required in this line?